tests/integration/targets/mso_schema_template_vrf_contract/tasks/main.yml

```diff
@@ -41,6 +41,15 @@
 #     - https://{{ apic_hostname }}
 #     state: present
 
```

Comment:
The Remove Schemas block has been shifted. What's the reason?

Reply:
As discussed this is required to make it work for the new cloud modules.